packages/fw-theme-builder/src/fw-theme-builder.ts

```diff
@@ -0,0 +1,313 @@
+import { html, css, LitElement } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import "./fw-color-pick";
+import "./fw-size-pick";
+import "./fw-font-pick";
+import { fontoptions, initialthemenew } from "./models";
+
+@customElement("fw-theme-builder")
+class FwThemeBuilder extends LitElement {
+  @state()
+  nav = "home";
+
+  @property({ type: Array })
+  fontOptions = fontoptions;
+
+  @property({ type: Object })
+  theme = initialthemenew;
+
+  @property({ type: Boolean })
+  viewByGroup = false;
+
+  static styles = css`
+    .back-button {
+      user-select: none;
+      cursor: pointer;
+      width: 3rem;
+      padding: 0.5rem 0.5rem;
+      z-index: 100;
+    }
+    .back-hidden {
+      opacity: 0;
+      cursor: default;
+    }
+    .back-icon {
+      height: 1rem;
+    }
+
+    .action-button {
+      transition: all 0.1s ease-in-out;
+      cursor: pointer;
+      width: 3.5rem;
+      user-select: none;
+      text-align: center;
+      border-radius: 4px;
+      font-family: "DM Sans", sans-serif;
+      padding: 0.5rem 0.5rem;
+      box-shadow: #1b1b1b3b 0px 4px 2px;
+    }
+    .save-btn {
+      background-color: #aaf16f;
+    }
+    .discard-btn {
+      background-color: #e7e7e7;
+    }
+    .action-button:active {
+      box-shadow: none;
+      transform: translateY(-2px);
+    }
+    .color-group-heading {
+      font-weight: 300;
+      margin: 0.25rem 0rem 0.1rem 0rem;
+    }
+    .section-heading {
+      font-weight: 300;
+      margin: 1rem 0rem 0.25rem 0rem;
+      border-bottom: 1px solid #1b1b1b3b;
+      padding-bottom: 0.2rem;
+    }
+  `;
+
+  sectionChangeHandler(e: any, s: string) {
+    this.nav = s;
+  }
+
+  navigateBack() {
+    if (this.nav == "home") return;
+    if (this.nav == "Colors" || this.nav == "Sizes" || this.nav == "Fonts") {
+      this.nav = "home";
+    } else {
+      this.nav = "Colors";
+    }
+  }
+
+  createFontPickComponent(font: string) {
+    return html` <fw-font-pick
+      exportparts="font-container, font-label, font-button, font-dropdown-container, font-dropdown-option, font-dropdown-selected"
+      .label=${font}
+      @value-changed=${(e: any) => {
+        let detail = {
+          type: font,
+          value: e.detail.value,
+        };
+        this.shadowRoot?.dispatchEvent(
+          new CustomEvent("font-change", {
```

Comment:
```suggestion
          new CustomEvent("font-changed", {
```
Same for other events